startcode.ps1

```diff
@@ -0,0 +1,12 @@
+﻿
+# In the case the .dotnet folder exists, need to assume it's possibly where the
+# dotnet sdk listed by global.json exists. Rather than parsing out global.json,
+# checking for matches in that directory (non-trivial), just put it first on the
+# path, enable mulit-level lookup and start code.
+$dotnetPath = Join-Path (Get-Location) ".dotnet"
+if (Test-Path $dotnetPath) {
+  $env:DOTNET_MULTILEVEL_LOOKUP=1
+  $env:PATH="$dotnetPath;$env:PATH"
+}
```

Comment:
This is inconsistent with `startvs.ps1` which always chooses `.dotnet\dotnet.exe`. Should we update the script to use multi-level lookup, as well?

Also `activate.ps1` always chooses `.dotnet` but that one might be expected since it's called "activate". But could also use multi-level lookup, not sure.

Reply:
Yeah I think that `startvs.ps1` should use the same approach too.